docs/reference-guides/filters/editor-filters.md

```diff
@@ -66,112 +209,38 @@ addFilter(
+function example_filter_block_editor_rest_api_preload_paths_when_post_provided( $preload_paths, $editor_context ) {
 	if ( ! empty( $editor_context->post ) ) {
 		array_push( $preload_paths, array( '/wp/v2/blocks', 'OPTIONS' ) );
 	}
 	return $preload_paths;
 }
-
-add_filter( 'block_editor_rest_api_preload_paths', 'wpdocs_filter_block_editor_rest_api_preload_paths_when_post_provided', 10, 2 );
 ```
 
-### Available default editor settings
-
-#### `richEditingEnabled`
-
-If it is `true` the user can edit the content using the visual editor.
-
-It is set by default to the return value of the [`user_can_richedit`](https://developer.wordpress.org/reference/functions/user_can_richedit/) function. It checks if the user can access the visual editor and whether it’s supported by the user’s browser.
-
-#### `codeEditingEnabled`
-
-Default `true`. Indicates whether the user can access the code editor **in addition** to the visual editor.
-
-If set to false the user will not be able to switch between visual and code editor. The option in the settings menu will not be available and the keyboard shortcut for switching editor types will not fire.
-
 ## Logging errors
 
-_**Note:** Since WordPress 6.1._
+A JavaScript error in a part of the UI shouldn't break the whole app. To solve this problem for users, React library uses the concept of an ["error boundary"](https://reactjs.org/docs/error-boundaries.html). Error boundaries are React components that catch JavaScript errors anywhere in their child component tree and display a fallback UI instead of the component tree that crashed.
 
-A JavaScript error in a part of the UI shouldn’t break the whole app. To solve this problem for users, React library uses a concept of an [“error boundary”](https://reactjs.org/docs/error-boundaries.html). Error boundaries are React components that catch JavaScript errors anywhere in their child component tree, and display a fallback UI instead of the component tree that crashed.
+The `editor.ErrorBoundary.errorLogged` action allows you to hook into the [Error Boundaries](https://reactjs.org/docs/error-boundaries.html) and gives you access to the error object.
 
-### `editor.ErrorBoundary.errorLogged`
-
-Allows you to hook into the [Error Boundaries](https://reactjs.org/docs/error-boundaries.html) and gives you access to the error object.
-
-You can use this action if you want to get hold of the error object that's handled by the boundaries, i.e to send them to an external error tracking tool.
-
-_Example_:
+You can use this action to get hold of the error object handled by the boundaries. For example, you may want to send them to an external error-tracking tool. Here's an example:
 
 ```js
+import { addAction } from '@wordpress/hooks';
+
 addAction(
 	'editor.ErrorBoundary.errorLogged',
 	'mu-plugin/error-capture-setup',
 	( error ) => {
-		// error is the exception's error object
+		// Error is the exception's error object.
```

Comment:
I understand that `ErrorCaptureTool.captureError` is an example of a (non-existing) tool that would track the error. Still, this example wouldn't work when copied and pasted to a real project.
I would be more explicit about this or would shape the example to replace `ErrorCaptureTool.captureError( error );` by `console.log( error );`

> You can use this action to get hold of the error object handled by the boundaries. For example, you may want to send them to an external error-tracking tool. Here's an example:

```
addAction(
	'editor.ErrorBoundary.errorLogged',
	'mu-plugin/error-capture-setup',
	( error ) => {
		// error is the exception's error object. 
		// we can console it or send it to an external error-tracking tool
		console.log ( error );
	}
);
```

Reply:
Good call. I update this in the most recent commit.